spec.html

```diff
@@ -12037,8 +12037,9 @@ <h1>Runtime Semantics: Evaluation</h1>
         <emu-grammar>SubstitutionTemplate : TemplateHead Expression TemplateSpans</emu-grammar>
         <emu-alg>
           1. Let _head_ be the TV of |TemplateHead| as defined in <emu-xref href="#sec-template-literal-lexical-components"></emu-xref>.
-          1. Let _sub_ be the result of evaluating |Expression|.
-          1. ReturnIfAbrupt(_sub_).
+          1. Let _subRef_ be the result of evaluating |Expression|.
+          1. ReturnIfAbrupt(_subRef_).
```

Comment:
this line isn't needed, since it's covered by step 1 of https://tc39.github.io/ecma262/#sec-getvalue